env.sh

```diff
@@ -0,0 +1,3 @@
+export SENDGRID_KEY=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
```

Comment:
This file should definitely be hidden! Perhaps in the `.gitignore`, or maybe just make this file private somehow!